src/Servers/IIS/IIS/test/IISExpress.FunctionalTests/OutOfProcess/NtlmAuthentationTest.cs

```diff
@@ -29,7 +29,8 @@ public static TestMatrix TestVariants
             => TestMatrix.ForServers(DeployerSelector.ServerType)
                 .WithTfms(Tfm.NetCoreApp30);
 
-        [ConditionalTheory(Skip = "https://github.com/aspnet/AspNetCore/issues/8329")]
+        [ConditionalTheory]
+        [Flaky("https://github.com/aspnet/AspNetCore/issues/8329")]
```

Comment:
These three. At this point I think you can just undo all changes to the src/Servers/IIS subfolder.